site/src/components/WorkspacesTable/WorkspacesTable.tsx

```diff
@@ -13,6 +13,7 @@ const Language = {
   version: "Version",
   lastBuilt: "Last Built",
   status: "Status",
+  lastBuiltBy: "By",
```

Comment:
I wonder if there are some columns we choose to hide on the workspaces table (but continue to display on the workspace view).

Reply:
Hmm, I am not sure if there is a piece of information we don't want, but maybe we can be minimalistic with the Version column -- have a check mark icon or a red/yellow warning icon, instead of the full `Up to date` or `Outdated` columns. 🤔

Reply:
Yeah, that might be nice in the future! Just an observation - no need to change anything.